model_compression_toolkit/core/common/quantization/quantization_config.py

```diff
@@ -13,13 +13,29 @@
 # limitations under the License.
 # ==============================================================================
 
-from dataclasses import dataclass, field
+from dataclasses import dataclass
 import math
 from enum import Enum
+from typing import Optional, Dict, NamedTuple, List
 
+from model_compression_toolkit import DefaultDict
 from model_compression_toolkit.constants import MIN_THRESHOLD
 
 
+class CustomOpsetLayers(NamedTuple):
+    """
+    This struct is used to define a set of operators to map to a custom operator set defined in the TPC.
+
```

Comment:
This struct is used to define a mapping between a set of framework operators to a custom opset name.